docs/content/en/contribute/general/github/_index.md

```diff
@@ -0,0 +1,276 @@
+     ```
+
+### Create a new branch and make your changes
+
+Now you have your upstream setup in your local machine.
+You need to create a local branch where you will make your changes.
+Be sure that your branch is based on and sync'ed with `main`,
+unless you intend to create a derivative PR:
+
+The following sequence of commands does that:
+
+```console
+git checkout main
+git pull upstream main
+git push origin main
+git checkout -b <my-new-branch>
+```
+
+Now you can make your changes, build them locally to check formatting,
+then create a PR to add these changes to the documentation set
+
+### Submitting new content through a pull request
+
+If you have forked and cloned the repository,
+you can modify the documentation or add new material
+by editing the markdown file(s) in the local clone of your fork
+and then submitting a *pull request (PR)*.
+
+Note that you can also modify the source using the GitHub editor.
+This is very useful when you want to fix a typo or make a small editorial change
+but, if you are doing significant writing,
+it is generally better to work on files in your local clone.
+
+1. Execute the following and check the output to ensure that your branch is set up correctly:
+
+   ```console
+   git status
+   ```
+
+1. Do the writing you want to do in your local branch, checking the formatted version at `localhost:1314`.
+
+1. When you have completed the writing you want to do, close all files in your branch and run `git status` to confirm
+   that it correctly reflects the files you have modified, added, and deleted.
+
+1. Add and commit your changes.  Here, we commit all modified files but you can specify individual files to the
+   `git add` command.
+The `git commit -s` command commits the files and signs that you are contributing this intellectual property to the
+Keptn project.
+
+   ```console
+   git add .
+   git commit -s
+   ```
+
+   Use vi commands to add a description of the PR
+   (should be 80 characters or less) to the commit.
+   The title text should be prefixed with `docs:`
+   to conform to our semantic commit scheme.
+   This title is displayed as the title of the PR in listings.
+   You can add multiple lines explaining the PR here but, in general,
+   it is better to only supply the PR title here;
+   you can add more information and edit the PR title
+   when you create the PR on the GitHub UI page.
+
+1. Push your branch to github:
+   * If you cloned your fork to use SSH, the command is:
+
+      ```console
+      git push --set-upstream origin <branch-name>
+      ```
+
+      > **Note**
+      You can just issue the `git push` command.
+      Git responds with an error message that gives you the full command line to use; you can copy this command and
+      paste it into your shell to push the content.
+
+   * If you cloned your fork to use HTTP, the command is:
+
+      ```console
+      git push <need options/arguments>
```

Comment:
Whoops!  This needs to be filled in and I'm not sure what it is off the top of my head.  But you may be able to find that info in the git docs -- holler if you need help.